aiocache/backends/memory.py

```diff
@@ -78,9 +78,10 @@ async def _delete(self, key, _conn=None):
         return self.__delete(key)
 
     async def _clear(self, namespace=None, _conn=None):
-        if namespace:
+        ns = namespace or self.namespace
```

Comment:
let's keep the name of the original variable

```
namespace = namespace or self.namespace
```

Reply:
Will do this on the other file too